agent/test/unit/com/thoughtworks/go/agent/AgentProcessParentImplTest.java

```diff
@@ -278,14 +275,10 @@ private File randomFile(final String pathname) {
     }
 
     private void createRandomFile(File agentJar, final String data) {
-        FileOutputStream output = null;
-        try {
-            output = new FileOutputStream(agentJar);
+        try (FileOutputStream output = new FileOutputStream(agentJar)) {
             IOUtils.write(data, output);
         } catch (IOException e) {
-            throw new RuntimeException(e);
-        } finally {
-            IOUtils.closeQuietly(output);
+            e.printStackTrace();
```

Comment:
I think it's ok to not wrap it around a runtime exception as the junit test where it's used has ```Exception``` on the signature.